mne/report.py

```diff
@@ -1556,6 +1562,54 @@ def _render_forward(self, fwd_fname):
                                         repr=repr_fwd)
         return html
 
+    def _render_sensitivity_map(self, fwd_fname, subject):
+        import matplotlib.pyplot as plt
+
+        """Render Sensitivity plots.
+        """
+        caption = u'Sensitivity : %s' % fwd_fname
+        div_klass = 'sensitivity'
+        img_klass = 'Sensitivity'
+        show = True
+
+        fwd = read_forward_solution(fwd_fname)
+        mag_map = sensitivity_map(fwd, ch_type='eeg', mode='fixed')
+        brain = mag_map.plot(subject=subject, time_label='Magnetometer sensitivity',
+                             hemi='rh', subjects_dir=subjects_dir)
+
+        html = []
+
+        fig = plt.figure()
+        plt.hist(mag_map.data.ravel(),
+                 bins=20, label=['Magnetometers'],
+                 color=['b'])
+        plt.title('Normal orientation sensitivity')
+        plt.xlabel('sensitivity')
+        plt.ylabel('count')
+        plt.legend()
+        global_id = self._get_id()
+        img = _fig_to_img(fig=fig)
+        html.append(image_template.substitute(img=img, id=global_id,
+                                         div_klass=div_klass,
+                                         img_klass=img_klass,
+                                         caption=caption,
+                                         show=show))
+
+        brain.save_image('sensitivity_mag.png')
```

Comment:
I think you do not need to do this. The logic is taken care of here: https://github.com/mne-tools/mne-python/blob/master/mne/report.py#L65-L68. You should be somehow able to get the figure handle for the mayavi figure ... maybe [this function](http://docs.enthought.com/mayavi/mayavi/auto/mlab_figure.html#gcf) will help you.